build-system/conformance-config.textproto

```diff
@@ -349,10 +349,11 @@ requirement: {
   whitelist: 'extensions/amp-date-picker/0.1/dates-list.js'
 }
 
-requirement: {
-  type: BANNED_PROPERTY_CALL
-  error_message: 'ChildNode.p.after() is unusual. Please use Node.p.insertBefore()'
-  value: 'Element.prototype.after'
-  value: 'DocumentType.prototype.after'
-  value: 'CharacterData.prototype.after'
-}
+# TODO(rsimha): Reenable when we move RRule to a node module.
```

Comment:
@jridgewell I think we need to wait for a closure compiler upgrade before moving to the `rrule` npm package.

Meanwhile, I've suggested an alternative fix in https://github.com/ampproject/amphtml/pull/22946#issuecomment-504217524